site/public/scss/_header.scss

```diff
@@ -315,6 +314,43 @@ header {
     }
 }
 
+.cookie__banner {
```

Comment:
It'd be better to have `left: 33%` and `right: 33%` so that it's guaranteed to be centered instead of `width: 33%`.

Also, let's have it at the bottom of the page since that's generally where cookie banners lie, so `bottom: 0` and remove `top: xxx`.

I think some more padding on the top and bottom would be nice as well